python/ray/ray_constants.py

```diff
@@ -150,20 +150,32 @@ def to_memory_units(memory_bytes, round_up):
 LOGGER_LEVEL_CHOICES = ["debug", "info", "warning", "error", "critical"]
 LOGGER_LEVEL_HELP = ("The logging level threshold, choices=['debug', 'info',"
                      " 'warning', 'error', 'critical'], default='info'")
+# Default param for RotatingFileHandler
+# maxBytes. 10G by default. We intentionally set the default value high
+# so that users who won't care don't know about the existence of this.
+LOGGING_ROTATE_BYTES = 10 * 1000 * 1000 * 1000
```

Comment:
Seems to me like the default should "just work" -- the defaults here are so large that we basically don't have log rotation. What's the concern with having more reasonable defaults?

Reply:
The main concern is some users who don't know anything about log rotation (nor requiring them) lose logs because of log rotation. I don't have strong opinion about this though. Do you have any suggested default?

Reply:
That seems preferable to me vs. users accidentally filling up their disk and crashing their applications, but I'm not totally sure. We can leave what you have for now as it's the most conservative change and consider changing it in the future.